tom.py

```diff
@@ -426,8 +428,13 @@ def main():
         log.basicConfig(level=numeric_level, format=fmt)
     else:
         log.basicConfig(format=fmt)
-
-    run_tom(args.interactive)
+    if args.continuous:
+        while True:
+            run_tom(args.interactive)
+            print("Iteration complete, sleeping for 12 seconds")
+            sleep(12)
```

Comment:
Why sleep for 12 seconds?

Reply:
Only in continuous mode, to avoid hiting rate limit on GitHub.